cip/cip30/src/main/java/com/bloxbean/cardano/client/cip/cip30/CIP30ValueDeserializer.java

```diff
@@ -0,0 +1,48 @@
+package com.bloxbean.cardano.client.cip.cip30;
+
+import co.nstant.in.cbor.CborDecoder;
+import co.nstant.in.cbor.model.*;
+import com.bloxbean.cardano.client.exception.CborDeserializationException;
+import com.bloxbean.cardano.client.transaction.spec.MultiAsset;
+import com.bloxbean.cardano.client.transaction.spec.Value;
+
+import java.util.List;
+
+/**
+ * Deserialize cbor from CIP30's wallet.getBalance()
+ */
+public class CIP30ValueDeserializer {
+
+    private CIP30ValueDeserializer() {
+        throw new IllegalStateException("Utility Class");
+    }
+
+    /**
+     * Deserialize cbor from CIP30's wallet.getBalance() method
+     * @param bytes balance cbor bytes from wallet.getBalance()
+     * @return {@link Value}
+     */
+    public static Value deserialize(byte[] bytes) throws CborDeserializationException {
```

Comment:
@edridudi If we move Value's deserialization code from TransactionOutput to Value class, do we still need this class? I understand why you have introduced this class now, but it may make sense to keep this logic in one place for easy maintenance in future. Because now there are 3 different ways value can be represented between Shelley to Babbage. For any change in future, it has to be done in both places.